tests/containers/helm.pm

```diff
@@ -109,6 +112,7 @@ sub run {
     assert_script_run("helm search repo apache");
     assert_script_run("helm show all $chart");
 
+    helm_cleanup() unless $self->{is_k3s};
```

Comment:
does it make sense that one line below there is variable  with same name but not from `$self` ?

Reply:
Good catch